org.hdfgroup.HDFView.yml

```diff
@@ -1,13 +1,14 @@
 app-id: org.hdfgroup.HDFView
 runtime: org.freedesktop.Platform
-runtime-version: "23.08"
+runtime-version: "24.08"
 sdk: org.freedesktop.Sdk
 sdk-extensions:
   - org.freedesktop.Sdk.Extension.openjdk
 build-options:
   env:
-    JAVA_HOME: /usr/lib/sdk/openjdk/jvm/openjdk-21
+    JAVA_HOME: /usr/lib/sdk/openjdk/jvm/openjdk-22
```

Comment:
Official builds seems to [have been built and tested with OpenJDK 21](https://github.com/HDFGroup/hdfview/releases/tag/v3.3.2). I guess it's ok to switch to 22 (and thus SDK  24.08) but would be nice to isolate that change is a separate commit.